pkg/raft/testdata/slow_follower_after_compaction.txt

```diff
@@ -96,24 +96,49 @@ compact 1 17
 ----
 1/18 EntryNormal "prop_1_18"
 
-# Trigger a round of empty MsgApp "probe" from leader. It will reach node 3
-# which will reply with a rejection MsgApp because it sees a gap in the log.
-# Node 1 will reset the MsgApp flow and send a snapshot to catch node 3 up.
+# Trigger a heartbeat timeout to allow the leader to detect that it can't send a
+# MsgApp to node 3 because it wil get rejected. Instead, the leader should send
+# a snapshot.
```

Comment:
"because it will get rejected" -> "because the relevant part of the log is already compacted".